lib/src/drawings.dart

```diff
@@ -69,6 +72,7 @@ class TextDrawing extends GYWDrawing {
   /// The text size. Overrides the font size.
   final int? size;
 
+  // TODO(roscale): It doesn't make sense to have nullable color.
   /// The color of the text (in 8-characters ORGB format)
   final String? color;
```

Comment:
Indeed, you can fix that, and you can use a `Color` object from flutter material. Be careful not to change the way colors are interpreted in the JSON.